File: ObjectOrientation/ColorSample_base/app/src/main/java/com/example/colorsample/BlueActivity.java
package com.example.colorsample;

import android.graphics.Color;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

public class BlueActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.layout).setBackgroundColor(Color.BLUE);

        findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
        // 別のActivityに遷移する時の処理イメージ
//                Intent intent = new Intent(BlueActivity.this, YellowActivity.class);
//                intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
//                startActivity(intent);
            }
        });
    }
}

File: ObjectOrientationKotlin/ObjectOrientationKotlin_base/app/src/main/java/com/eugo/objectorientationkotlin/BlueActivity.kt
package com.eugo.objectorientationkotlin

import android.graphics.Color
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class BlueActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<View>(R.id.layout).setBackgroundColor(Color.BLUE)
        findViewById<View>(R.id.button).setOnClickListener {
            // 別のActivityに遷移する時の処理イメージ
//            val intent = Intent(this, YellowActivity::class.java)
//            intent.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP
//            startActivity(intent)
        }
    }
}